e2e_playwright/playwright.config.ts

```diff
@@ -0,0 +1,33 @@
+import 'dotenv/config'
```

Comment:
This doesn't seem to be loading in the env vars for me automatically.  When I run the tests I'm getting `Error: Credentials missing, have you set the correct environment variables?`

Reply:
🤦‍♂️  I hadn't copied the new vars from `.env.local` to `.env`